terraform/test/fixtures/bot_0_create.tf

```diff
@@ -0,0 +1,26 @@
+locals {
+  bot_name = "test"
+}
+
+resource "teleport_provision_token" "bot_test" {
+  metadata = {
+    expires = "2028-01-01T00:00:00Z"
+    name = "bot-test"
+  }
+
+  spec = {
+    roles = ["Bot"]
+    bot_name = local.bot_name
+    join_method = "token"
+  }
+}
+
+resource "teleport_bot" "test" {
+  name = local.bot_name
+  token_id = "bot-test"
```

Comment:
I'm a bit bothered that using a reference here fails in CI but works locally, I'd ideally prefer to have:

```terraform
resource "teleport_bot" "test" {
  name = local.bot_name
  token_id = teleport_provision_token.bot_test.metadata.name
  roles = ["terraform"]
}
```
... but CI fails every time with:

```
    bot_test.go:45: Step 1/4 error: Error running pre-apply refresh: exit status 1
        
        Error: Unsupported attribute
        
          on terraform_plugin_test.tf line 29, in resource "teleport_bot" "test":
          29:   token_id = teleport_provision_token.bot_test.metadata.name
        
        This value does not have any attributes.
```

Maybe an ordering issue or something? Hrm.